bgpd/bgp_zebra.c

```diff
@@ -1382,6 +1397,24 @@ void bgp_zebra_announce(struct bgp_dest *dest, const struct prefix *p,
 			ifindex_t ifindex = IFINDEX_INTERNAL;
 			struct in6_addr *nexthop;
 
+			if (bgp->table_map[afi][safi].name) {
```

Comment:
Do we need this to be applied two times in a row? I see this already handled above, maybe can be called once?

Reply:
Ok, will fix it.